### PR TITLE
LAU-898 email validation

### PR DESCRIPTION
**Pull request by priyankaVerma21**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/LAU-898

### Change description ###
This change is for validation on email

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/views/logons/template.njk [hunk 1]
```diff
@@ -42,7 +42,12 @@
         classes: "govuk-!-width-three-quarters govuk-!-margin-top-1",
         id: "emailAddress",
         name: "emailAddress",
-        value: logonForm.emailAddress
+        value: logonForm.emailAddress,
+        type: "email",
+        spellcheck: false,
+        hint: {
+          text: "We’ll use this to search the users"
```
Comment (MartinYSpasov) [inline src/main/views/logons/template.njk:49]: Do we need hint?

File: src/main/views/user-deletions/template.njk [hunk 2]
```diff
@@ -44,7 +44,12 @@
         classes: "govuk-!-width-three-quarters govuk-!-margin-top-1",
         id: "emailAddress",
         name: "emailAddress",
-        value: deletedUsersForm.emailAddress
+        value: deletedUsersForm.emailAddress,
+        type: "email",
+        spellcheck: false,
+        hint: {
+          text: "We’ll use this to search the users"
```
Comment (MartinYSpasov) [inline src/main/views/user-deletions/template.njk:51]: Do we need hint?

**Review by MartinYSpasov — CHANGES_REQUESTED** [review]

File: src/main/views/logons/template.njk (hunk 1 above)
Comment (priyankaVerma21) [reply src/main/views/logons/template.njk:49]: As per govuk Email example , it is best to say why we are asking for emails

Comment (priyankaVerma21) [reply src/main/views/logons/template.njk:49]: https://design-system.service.gov.uk/patterns/email-addresses/

File: src/main/views/user-deletions/template.njk (hunk 2 above)
Comment (priyankaVerma21) [reply src/main/views/user-deletions/template.njk:51]: as per thus https://design-system.service.gov.uk/patterns/email-addresses/

**Review by MartinYSpasov — APPROVED** [review]

**Pull request merged**